frontend/src/lib/components/PropertyFilters/components/PropertyFiltersDisplay.tsx

```diff
@@ -11,8 +11,14 @@ const PropertyFiltersDisplay: React.FunctionComponent<Props> = ({ filters, style
     return (
         <div className="PropertyFilters mb-4" style={style}>
             {filters &&
-                filters.map((item) => {
-                    return <PropertyFilterButton key={item.key} item={item} />
+                filters.map((item, idx) => {
+                    return (
+                        <>
+                            {' '}
+                            <PropertyFilterButton style={{ margin: '0.1rem' }} key={item.key} item={item} />{' '}
+                            {idx !== filters.length - 1 ? ',' : ''}
```

Comment:
From experience with our `PropertyFilter` I think this is the riskiest part of the change

I'd rather wrap an array of `PropertyFilter` in a `SentenceList` in the specific case(s) where they need to have commas so we remove the risk of commas in unexpected places

This is way more a criticism of our (lack of) visual testing of such a critical component than your PR!

Reply:
> This is a solid change... just the one bit we could do with tweaking 🙌

Here's what it looks like now! @pauldambra 
<img width="200" alt="Screen Shot 2022-08-12 at 11 37 39 AM" src="https://user-images.githubusercontent.com/17347501/184390263-409b80f0-d0d2-4a8a-a444-fb223f2286d2.png">